crates/red_knot_python_semantic/resources/mdtest/binary/instances.md

```diff
@@ -259,11 +259,7 @@ class A:
 class B:
     __add__ = A()
 
-# TODO: this could be `int` if we declare `B.__add__` using a `Callable` type
```

Comment:
We can also get rid of the `Unknown` by declaring it as `A` (`__add__: A = A()`), so I don't think it's worthy a TODO. The test is fine as is, I think.